models/category.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable prettier/prettier */
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const categorySchema = new Schema({
+    date: { type: Date, default: Date.now },
+    username: { type: String, required: true },
+    category: { type: String, required: true }
+});
```

Comment:
Need to define a relationship between category and notes
see: [this activity from week 18]( https://ucdavis.bootcampcontent.com/ucdavis-boot-camp/UCD-SAC-FSF-PT-09-2019-U-C/blob/master/Monday_Wednesday/18-mongo-mongoose/01-Activities/18-Intro-Populate/models/Library.js)

Suggestion:
```suggestion
const categorySchema = new Schema({
    date: { type: Date, default: Date.now },
    username: { type: String, required: true },
    category: { type: String, required: true },
    notes: [
    {
      type: Schema.Types.ObjectId,
      ref: "Note"
    }
  ]

});
```